Assets/GothicVR/Scripts/UI/TurnSettingDropdownController.cs

```diff
@@ -29,7 +29,7 @@ void Awake()
         {
             dropdown.options.Add(new TMP_Dropdown.OptionData() { text = item });
         }
-        dropdown.itemText.font = GameData.I.GothicSubtitleFont;
+        dropdown.itemText.font = GameData.I.EmptyFont;
```

Comment:
The more I see it, the more I think "Empty font" is there ring name. We need to get it by a specific name like GetMenuFont(). However G1 is handling it. We want it as well. 😚